components/shared/Button.js

```diff
@@ -4,10 +4,10 @@ import styled from "styled-components";
 const Button = styled.button`
   font-family: Roboto, sans-serif;
   font-size: ${({ fontSize }) => fontSize}px;
-  font-weight: ${({ secondary }) => (secondary ? 300 : 600)};
+  font-weight: ${({ secondary }) => (secondary ? 500 : "bold")};
```

Comment:
`bold` is a 700 in css, but it's not connected in the head `https://fonts.googleapis.com/css?family=Roboto:300,400,500`